contracts/src/Assets.sol

```diff
@@ -95,17 +95,14 @@ library Assets {
 
     /// @dev Enqueues a create native token message to substrate.
     /// @param token The ERC20 token address.
-    function registerToken(address token, bytes2 createTokenCallID)
-        external
-        returns (bytes memory payload, uint256 extraFee)
-    {
```

Comment:
Please don't make any formatting/whitespace changes. If necessary, we should disable any auto-formatters until the audit is complete.

This will help the auditors.

Reply:
https://github.com/Snowfork/snowbridge/pull/961#discussion_r1334933344